docs/modules/operation/pages/provisioning/auto-discovery.adoc

```diff
@@ -72,3 +72,98 @@ A definition without any detectors will fall back to ICMP ping for discovery.
 
 </discovery-configuration>
 ----
+
+[[ga-manual-discovery]]
+== Manual discovery configuration
+
+In addition to the configuration above, there are some optional settings that can be configured via `$\{OPENNMS_HOME}/etc/discovery-configuration.xml`.
+
+.Discovery configuration parameters
+[options="header" cols="1,1,3"]
+|===
+| Name
+| Default
+| Description
+
+| packets-per-second
+| 1
+| This is the number of ICMP packets that will be generated each second.
+Note that there is a relationship between the packets-per-second and the number of threads.
+If your network has an average latency of 500ms, then setting packets-per-second to 2 would double the speed at which NewSuspect messages were created.
+But if there is only one thread available, setting this number to 3 would have little effect - the single thread would be processing as many packets as it could as fast as it could.
```

Comment:
```suggestion
But if there is only one thread available, setting this number to 3 would have little effect: the single thread would process as many packets as it could as fast as it could.
```

Reply:
But if there is only one thread available, setting this number to `3` would have little effect.
The single thread would process as many packets as it could as fast as it could.
```